src/io/batchify.cc

```diff
@@ -150,7 +150,7 @@ class StackBatchify : public BatchifyFunction {
         }
         int sbs = static_cast<int>(bs);
         MSHADOW_TYPE_SWITCH_WITH_BOOL(dtype, DType, {
-          _Pragma("omp parallel for num_threads(bs)")
+          #pragma omp parallel for num_threads(bs)
```

Comment:
use `__pragma` for msvc if it does not support `_Pragma`

Reply:
We can unify the macro like
```c++
#ifndef _MSC_VER
#define __pragma(id) _Pragma(#id)
#endif
```